### PR TITLE
changelog: `conflict()` and `file()` deprecations are yet unreleased.

### PR DESCRIPTION
**Pull request by essiene**
Move this update under `Unreleased`. I started the change before the last release and after rebasing, forgot to move it. Fixing it now thanks to @yuja catching this on time.

Issue: #4122
# Checklist

If applicable:
- [x] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * The `git.push-branch-prefix` config has been deprecated in favor of
   `git.push-bookmark-prefix`.
 
+* `conflict()` and `file()` revsets have been renamed to `conflicts()` and `files()`
+  respectively. With this, all revsets that could potentially return multiple
+  results are named in the plural.
```
Comment (martinvonz) [inline CHANGELOG.md:39]: I'm not sure I understood @yuja's point correctly, but I think he was also saying that the last sentence isn't quite correct. We have e.g. `description()` and `author()` that are not named in the plural.

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,6 +34,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
+
+
```
Comment (yuja) [inline CHANGELOG.md:41]: nit: excessive blank lines

**Review by yuja — APPROVED** [review]
Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (essiene) [reply CHANGELOG.md]: I've just removed that sentence. The deprecation notice is enough by itself. The issue discussions have more detail than I was trying to cram into one ambigous and wrong sentence. thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (essiene) [reply CHANGELOG.md:39]: Removed. Thanks.

**Review by martinvonz — APPROVED** [review]

**Pull request merged**